json-fleece-swagger2/json-fleece-swagger2.cabal

```diff
@@ -19,53 +19,6 @@ tested-with:
 extra-source-files:
     examples/uber/codegen.dhall
     examples/uber/uber.json
-    examples/uber/Uber/Operations/Estimates/Price.hs
-    examples/uber/Uber/Operations/Estimates/Price/EndLatitude.hs
-    examples/uber/Uber/Operations/Estimates/Price/EndLongitude.hs
-    examples/uber/Uber/Operations/Estimates/Price/StartLatitude.hs
-    examples/uber/Uber/Operations/Estimates/Price/StartLongitude.hs
-    examples/uber/Uber/Operations/Estimates/Time.hs
-    examples/uber/Uber/Operations/Estimates/Time/CustomerUuid.hs
-    examples/uber/Uber/Operations/Estimates/Time/ProductId.hs
-    examples/uber/Uber/Operations/Estimates/Time/StartLatitude.hs
```

Comment:
I don't think these should be removed?

Reply:
This was just a mistake, we're putting these back in when I merge Tyler's branch into mine.